xblock/test/test_mixins.py

```diff
@@ -278,8 +289,27 @@ def test_add_xml_to_node(self):
             {
                 'field': 'Value1',
                 'simple_default': 'Value2',
-                'simple_default_with_force_export': 'Value3',
+                'force_export': 'Value3',
                 'unique_id_default': 'Value4',
-                'unique_id_default_with_force_export': 'Value5',
+                'unique_id_force_export': 'Value5',
             }
         )
+
+    @ddt.data(
+        (None, {'datetime': ''}),
```

Comment:
Nit: Maybe a `('', {'datetime': ''}),` here as well? Or is an empty string not allowed?